server/package.json

```diff
@@ -56,6 +57,7 @@
     "@types/supertest": "^6.0.0",
     "@typescript-eslint/eslint-plugin": "^6.0.0",
     "@typescript-eslint/parser": "^6.0.0",
+    "dotenv": "^16.4.5",
```

Comment:
I think we don't need `dotenv` as we already installed `@nestjs/config`
https://docs.nestjs.com/techniques/configuration

And  I'm not sure where `"@nestjs/mapped-types` is used! If you don't used this anywhere, could you delete it?